server/dialect-daco/pom.xml

```diff
@@ -30,8 +30,8 @@
     <properties>
         <maven.compiler.source>8</maven.compiler.source>
         <maven.compiler.target>8</maven.compiler.target>
-        <maven.antlr4.plugin.version>4.7</maven.antlr4.plugin.version>
-        <antlr.version>4.7.1</antlr.version>
+        <maven.antlr4.plugin.version>4.13.2</maven.antlr4.plugin.version>
+        <antlr.version>4.13.2</antlr.version>
```

Comment:
- [ ]  Why do we need to update the versions in other extensions?

coz, of the usecase tests. The generated usecaseTestLexer expects the version to be 4.13.2
```java
static { RuntimeMetaData.checkVersion("4.13.2", RuntimeMetaData.VERSION); }
```